es-app/src/views/gamelist/ISimpleGameListView.cpp

```diff
@@ -8,7 +8,7 @@
 #include "CollectionSystemManager.h"
 
 ISimpleGameListView::ISimpleGameListView(Window* window, FileData* root) : IGameListView(window, root),
-	mHeaderText(window), mHeaderImage(window), mBackground(window)
+	mHeaderText(window), mHeaderImage(window), mBackground(window), mThemeExtras(window)
```

Comment:
The `ThemeExtras` class was removed in the time since this grid view implementation was originally done.  This change needs to be reverted.